src/components/NavBar/NavBar.module.css

```diff
@@ -0,0 +1,35 @@
+.navBar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 16px;
+    background-color: #d4d4d4;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+    width: 100%;
+    border-radius: 10px;
+    margin-right: 10px;
+    height: 70px;
+    box-sizing: border-box;
+  }
+  
+.navList {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
```

Comment:
Consider adding these 2 to the index.css file- you don't have to, but it might help you avoid writing this 2 lines over and over again across you code.

Reply:
maybe I have a gap in my understanding but many time the element runs this over by using a default padding and margin. there I did found that necessary to add some of my own.

Reply:
We talked about the gap, please change :)